sql-statements/sql-statement-create-index.md

```diff
@@ -128,9 +128,62 @@ CREATE UNIQUE INDEX c1 ON t1 (c1);
 Query OK, 0 rows affected (0.31 sec)
 ```
 
+<<<<<<< HEAD
```

Comment:
@qiancai Please resolve the conflicts. Thanks!

Reply:
Done.